docs/source/nn.rst

```diff
@@ -341,11 +341,24 @@ From the ``torch.nn.utils`` module
     parameters_to_vector
     vector_to_parameters
 
+.. autosummary::
+    :toctree: generated
+    :nosignatures:
+
+    parametrize.register_parametrization
+    parametrize.remove_parametrizations
+    parametrize.cached
+    parametrize.is_parametrized
+    parametrize.set_caching
```

Comment:
unsymmetric naming... maybe `add_caching`/`remove_caching`. or `enable_caching`/`disable_caching`?

Reply:
With respect to what is this naming not symmetric?

Also, if I had to choose between those two, I would go with `add` / `remove`, as the other one might suggest to be a low-level API for the `cached` context manager, which is not the case.